### ISSUE TITLE
type のご報告

### ISSUE THREAD

**Issue by sp-au-mu-nl** [YutaroOgawa/pytorch_tutorials_jp#18]
こんにちは。誤植と思われるところを見つけたのでご報告します。
0_4_buildmodel_tutorial_js.ipynb の nn.Softmax のところで、
採取的な値は
となっていますが、
最終的な値は
ではないでしょうか?
(原文を確認できなかったので、的外れかもしれませんが。)
